test/SignatureChecker.t.sol

```diff
@@ -45,13 +45,54 @@ contract AccountWithValidFunction {
     }
 }
 
+contract Verifier {
```

Comment:
I don't think this is needed at all since the only tests that uses it is testing this contract, which isn't part of `src`. Am I missing something?

Reply:
https://github.com/MZero-Labs/common/pull/8/commits/b08fe3cd46db0bcd272a1ce76470bca0d5089713